tests/query/arcana-repeat.scm

```diff
@@ -27,64 +27,6 @@
 	)
 )
 
-; Three parts, all three identical
-(ImplicationLink
-	(ListLink
```

Comment:
Nil, can you restore these tests, and just change ImplicationLink to some other ordered link type? The  link type here mostly does not matter,  what did matter was the structure of the patterns being explored. These tests are important; they're testing for some complex traversal scenarios.